core/src/main/java/org/apache/carbondata/core/statusmanager/LoadMetadataDetails.java

```diff
@@ -211,25 +234,32 @@ public long getLoadStartTimeAsLong() {
    * @return
    */
   private long convertTimeStampToLong(String factTimeStamp) {
```

Comment:
What is the difference between this function and getTimeStamp, seems they are doing the same thing

Reply:
This function getTimeStamp returns value in nano second and returns in mili second